File: C/unit_3/Fi.c
#include<stdio.h>
int main()
{  
   int l,b,ar,pr;
   printf("Enter the Lenght of the Rectangle : \n");
   scanf("%d",&l);
   printf("Enter the Breadth of the Rectangle : \n");
   scanf("%d",&b);
   ar = l*b;
   pr = 2*(l+b);
   if(ar>pr)
    printf("The Area is Greater than it's Perimeter");
   else
    printf("The Area is Smaller than it's Perimeter");
   return 0;
}

File: Java/unit_3/Fi.java
package Java.unit_3;

import java.util.Scanner;

public class Fi {
    public static void main(String[] args) {
        int l, b, ar, pr;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the Length of the Rectangle : \n");
        l = sc.nextInt();
        System.out.println("Enter the Bredth of the Rectangle : \n");
        b = sc.nextInt();
        ar = l * b;
        pr = 2 * (l + b);
        if (ar > pr)
            System.out.println("Area is Greater than its Perimeter !!!");
        else
            System.out.println("Area is Less than the Perimeter !!!");
        sc.close();
    }
}
